debian.Dockerfile

```diff
@@ -0,0 +1,93 @@
+FROM ubuntu:18.04
+ENV LIB_INSTALL_PREFIX ${LIB_INSTALL_PREFIX:-/usr}
+ENV OPENVAS_DEB_BUILD_DIR ${OPENVAS_DEB_BUILD_DIR:-/tmp/openvas}
+ENV OPENVAS_SCANNER_DEB_BUILD_DIR ${OPENVAS_SCANNER_DEB_BUILD_DIR:-/tmp/openvas-scanner}
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && apt-get install -q -y --fix-missing \
+  autotools-dev \
+  bison \
+  build-essential \
+  devscripts \
+  debhelper \
+  clang \
+  clang-format \
+  cmake \
+  gcc \
+  pkg-config \
+  libglib2.0-dev \
+  libgcrypt20-dev \
+  libgnutls28-dev \
+  libgpgme-dev \
+  libksba-dev \
+  libnet1-dev \
+  libpcap-dev \
+  libsnmp-dev \
+  libhiredis-dev \
+  libunistring-dev \
+  libxml2-dev \
+  tar \
+  uuid-dev \
+  wget \
+  libssh-gcrypt-dev && \
+	apt-get clean && \
+	rm -rf /var/lib/apt/lists/*
+#RUN apt-cache policy gnutls-bin && false
+# GnuTLS 3.5.18 is the default version available on Ubuntu 18.04, however openvas 21.4 requires at least 3.6.4
```

Comment:
Added the comment, as to why building `GnuTLS` was built from source. This is bit time consuming when building packages, but could not find any other way to bring newer version of GnuTLS on Ubuntu 18.04.